code/modules/client/preference/link_processing.dm

```diff
@@ -668,12 +668,7 @@
 					else if(S.bodyflags & HAS_ICON_SKIN_TONE)
 						var/const/MAX_LINE_ENTRIES = 4
 						var/prompt = "Choose your character's skin tone: 1-[S.icon_skin_tones.len]\n("
-						for(var/i = 1 to S.icon_skin_tones.len)
-							if(i > MAX_LINE_ENTRIES && !((i - 1) % MAX_LINE_ENTRIES))
-								prompt += "\n"
-							prompt += "[i] = [S.icon_skin_tones[i]]"
-							if(i != S.icon_skin_tones.len)
-								prompt += ", "
+						prompt += "Light to Dark"
 						prompt += ")"
```

Comment:
You can remove the "(" and ")" and just move the "Light to Dark" string inside the prompt string. 

There are a few places that modify skintone like this (mutation_powers.dm, appearance_changer.dm). You can probably go through and modify those to reference this as well if this is what you want to do.